docs/examples/exporters/GLTFExporter.html

```diff
@@ -90,6 +98,24 @@ <h3>[method:null parse]( [param:Object3D input], [param:Function onCompleted], [
 		Generates a .gltf (JSON) or .glb (binary) output from the input (Scenes or Objects)
 		</p>
 
+		<h3>[method:null parseChunks]( [param:Object3D input], [param:Function onCompleted], [param:Function onError], [param:Object options] )</h3>
+		<p>
+		[page:Object input] — Scenes or objects to export.<br />
+		[page:Function onCompleted] — Will be called when the export completes. The argument will be the generated glTF JSON or binary ArrayBuffer.<br />
+		[page:Function onError] — (optional) A function to be called if an error occurs during exporting. The function receives error as an argument.<br />
+		[page:Options options] — Export options see <em>.parse</em><br />
+		</p>
+		<p>
+		Returns an object with the following properties: <br />
+		<ul>
+			<li>json — object. The glTF's JSON content.</li>
+			<li>bin — Blob. The glTF's .bin file content. Or <em>undefined</em> if <em>options.binary</em> is set.</li>
+			<li>binFileName — string. The glTF's .bin file name, saved in the glTF's JSON content. Or <em>undefined</em> if <em>options.binary</em> is set. This property can be set with <em>options.binFileName</em></li>
+			<li>externalImageUrls — Array<[page:String String]>. The urls of the externally referenced images in the glTF file.</li>
```

Comment:
So `externalImageUrls` would always be an absolute HTTP(S) URL? I kind of feel like we should move away from using those, and just embed the image. Another option would be:

```
{
  'foo.bin': Blob,
  'foo.png': Blob,
  ...
}
````

Absolute URLs seem to mostly cause problems on export (e.g. https://github.com/aframevr/aframe-inspector/issues/547), and from the spec:

> Client implementations are required to support only embedded resources and relative external references (in a sense of RFC3986). Clients are free to support other schemes (such as http://) depending on expected usage.

Reply:
I like the object with the file name to image data mapping. Maybe the value could be either a `HTMLImageElement` or `Blob` depending on whether the texture uses an image or Canvas.
 
Currently absolute urls are exported. This change lets you export with relative urls in the glTF and download those images with the correct file names.